packages/languages/src/browser/language-client-contribution.ts

```diff
@@ -116,6 +133,10 @@ export abstract class BaseLanguageClientContribution implements LanguageClientCo
         };
     }
 
+    protected get workspaceContains(): Array<string> {
```

Comment:
Can we use a real method instead of a property getter here? Also why not `string[]` instead of `Array<string>`?

Reply:
it won't be symmetric to `documentSelector` and `globPatterns`.

Reply:
> it won't be symmetric to documentSelector and globPatterns.

👍 I see. What about the `Array<string>`? That is still valid, right?